instagram-app/src/components/PostContainer/PostHead.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import './Post.css';
+
+// Post images and username
+
+const PostHead = props => {
+    console.log("PostHead", props)
+    return (
+        <div className="postHeader">
+            <div className="postThumbWrap">
+            <img alt="postHead" src={props.thumbnailUrl} className="postThumb"/>
```

Comment:
the image is passed down as `props.thumbnailImg` instead of `props.thumbnailUrl`